### ISSUE TITLE
Sectigo Certificate Manager DNS validation records trigger label underscore warning

### ISSUE THREAD

**Issue by misilot** [StackExchange/dnscontrol#829]
Sectigo who provides certificate services for InCommon institutions utilize  record labels that start with an underscore. Uses a similar setup like AWS acm-validation in #519.

```
CNAME('_RANDOMSTRING', 'STRING1.STRING2.sectigo.com.'),
```

```
WARNING: label _RANDOMSTRING.example.net contains an underscore
```

Taking from the example @tlimoncelli posted, a possible solution would be

No warning if:

1. The .Name starts with _
2. and... The .Type is "CNAME"
3. and... the .Target ends with sectigo.com

Would it possible make sense to create an array similar to the prefix list in https://github.com/StackExchange/dnscontrol/blob/master/pkg/normalize/validate.go#L90-L100.

and update the logic in https://github.com/StackExchange/dnscontrol/blob/master/pkg/normalize/validate.go#L132 to check the array?

**Comment by tlimoncelli** [comment]
Rather than manage an ever-growing list, I think it would be best to just not warn if the label starts with a `_`.

Originally this was trying to warn against labels like "host_123" when the user meant "host-123" (a typical first-timer mistake).    I think that if you start something with `_` you already know what you are doing :-)

Thoughts?

**Comment by misilot** [comment]
I think that makes sense. I put together #830 which I think would resolve this.

**Comment by misilot** [comment]
Closing this since it was resolved with #830.
